src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -582,19 +582,25 @@ try
 
     RETURN_HR_IF(E_NOT_VALID_STATE, fontSize.area() == 0); // either dimension = 0, area == 0
 
-    if (this->_singleClickTouchdownPos)
+    // This is a copy of ControlInteractivity::PointerMoved
+    if (_singleClickTouchdownPos)
     {
-        const auto& touchdownPoint{ *this->_singleClickTouchdownPos };
-        const auto distance{ std::sqrtf(std::powf(cursorPosition.x<float>() - touchdownPoint.x<float>(), 2) + std::powf(cursorPosition.y<float>() - touchdownPoint.y<float>(), 2)) };
-        if (distance >= (std::min(fontSize.width(), fontSize.height()) / 4.f))
+        const auto touchdownPoint = *_singleClickTouchdownPos;
+        const auto dx = cursorPosition.x - touchdownPoint.x;
+        const auto dy = cursorPosition.y - touchdownPoint.y;
+        const auto w = fontSize.width;
+        const auto distanceSquared = dx * dx + dy * dy;
+        const auto maxDistanceSquared = w * w / 16; // (w /4)^2
+
+        if (distanceSquared >= maxDistanceSquared)

```

Comment:
No. Unless I made a weird mistake, the functionality is exactly the same. 🙂
The change is necessary since the old code wouldn't compile otherwise. Since a change was necessary I changed the distance check to a simpler standard distance check. It's rather unusual to use sqrt for that to be honest.
It's safer to just square the other side of the equation - no NANs & no explicit static_cast casts (which would've been necessary here).